### PR TITLE
Clean-up and release integration.

### PR DESCRIPTION
**Pull request by TheRealVira**
Taking inspiration of https://github.com/vigonotion/hass-simpleicons/tree/master this will help with releases and code clean-ups. Ref. https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/issues/27

### CODE REVIEW

**Comment by TheRealVira** [comment]
Didn't see it first, but this would also include part of https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/pull/14

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (DarwinsBuddy) [inline .vscode/launch.json:2]: please do not commit comments in json :)

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "version": "0.2.0",
+    "configurations": [
+      {
+        // Example of attaching to local debug server
```
Comment (DarwinsBuddy) [inline .vscode/launch.json:6]: please do not commit comments in json :)

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        "host": "localhost",
+        "pathMappings": [
+          {
+            "localRoot": "${workspaceFolder}",
+            "remoteRoot": "."
+          }
+        ]
+      },
+      {
+        // Example of attaching to my production server
```
Comment (DarwinsBuddy) [inline .vscode/launch.json:20]: please do not commit comments in json :)

File: .devcontainer/devcontainer.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+// See https://aka.ms/vscode-remote/devcontainer.json for format details.
```
Comment (DarwinsBuddy) [inline .devcontainer/devcontainer.json:1]: please do not commit comments in json :)

**Review by DarwinsBuddy — CHANGES_REQUESTED** [review]
Wow, you're a beast.
I created that ticket last week and 4 days later we got a cleanup and a release process.
Impressive!

If you could do one more thing?
I read that workflow and like the idea of separating you did. 

Could you document this approach with some instructions how to run the devcontainer etc. in a separate CONTRIBUTING.md (like here https://github.com/redhat-developer/vscode-yaml/blob/main/CONTRIBUTING.md) besides the README.md and refer to it there?

**Comment by DarwinsBuddy** [comment]
@TheRealVira I tried to contact you, but you make it really a challenge. :smile:  No email, no keybase, no nothing.

If you're interested furthermore on this project, either write me an email (christoph.spoerk@gmail.com) and/or create a keybase account for a secure channel: https://keybase.io/app

Have a nice weekend.
Cheers!

**Comment by TheRealVira** [comment]
Hey @DarwinsBuddy I'll get back to you as soon as I can. For now, please check out my LinkedIn: 

https://at.linkedin.com/in/ruehrig-j

I'll aso implement the suggested changes - no problem :D I love to support local projects, especially if they have the capability to make a great impact. With WNSM I can really see how this integration could support people saving energy and money for "free".

**Comment by DarwinsBuddy** [comment]
I found your profile already, though I am not having premium, thus I cannot send you a message. :grimacing:

**Comment by TheRealVira** [comment]
Wtf since when do you need LinkedIn Premium for messages.. whatever, here is my e-mail address:

therealvira@gmail.com

I'll also update my GitHub profile Readme to display my e-mail address.

**Comment by TheRealVira** [comment]
Anything else needed from my side? @DarwinsBuddy

**Comment by DarwinsBuddy** [comment]
hi sry. much to do in the office the last weeks. I hope I'll get to it tomorrow 🤞🤞🤞

**Pull request merged**